app/components/PlaylistView/index.js

```diff
@@ -102,16 +94,51 @@ class PlaylistView extends React.Component {
     );
   }
 
-  render() {
+  renderPlaylistTracksHeader () {
+    return (
+      <thead>
+        <tr>
+          <th>
+            <FontAwesome name='photo' />
+          </th>
+          <th>Artist</th>
+          <th>Title</th>
+        </tr>
+      </thead>);
+  }
+
+  render () {
     let { playlist } = this.props;
     return (
       <div className={styles.playlist_view_container}>
         <div className={styles.playlist}>
           {this.renderPlaylistInfo()}
           <div className={styles.playlist_tracks}>
-            {playlist.tracks.map((track, i) =>
-              this.renderPlaylistTrack(track, i)
-            )}
+            <table>
+              {this.renderPlaylistTracksHeader()}
+              <tbody>
+                {playlist.tracks.map((track, index) => {
+                  _.set(track, 'artist.name', track.artist);
```

Comment:
Dangerous, this will mutate the props and interfere with Redux. If you want to change the props, best clone them before doing that. Also, why are you doing that here?

Reply:
The original idea about track metadata was this:
* All data originates with a Discogs search. Initial track data is copied verbatim from what Discogs returns
* The above is augmented with lastfm data, by adding a "lastfm" key that contains data copied from lastfm verbatim
* As a track is added to the play queue, it is further augmented with stream data, returned by appropriate plugins

Reply:
I understand. But for instance, for Track Search, we do not rely on discogs but on lastfm, so the data doesn't have exactly the same format...

Reply:
Ok, for now let's just get rid of instances where global state is mutated, and we can worry about a uniform track class later.

Reply:
Simply clone the object from the redux state before changing its properties.
So e.g. instead of:
`_.set(track, 'artist.name', track.artist);`

do something like this:

```javascript
const newTrack = _.cloneDeep(track);
_.set(newTrack, 'artist.name', newTrack.artist);
```

Or use `Object.assign` to clone the object, similar to how it's done in all the reducers.